typescript/cli/src/utils/env.ts

```diff
@@ -0,0 +1,7 @@
+import z from 'zod';
+
+const envScheme = z.object({
+  HYP_KEY: z.string(),
+});
+
+export const ENV = envScheme.parse(process.env);
```

Comment:
I wonder if we should use `safeParse` here instead with a fallback to `{}`. Just wondering if random crap in the env object could cause an error. Maybe not though, just a passing thought

Reply:
Agreed. Added along with optional